### PR TITLE
Meghan-Harris_blog-entry

### PR DESCRIPTION
**Pull request by Meghansaha**
Not sure if adding mastodon is accepted? Added it just in case.

### CODE REVIEW

File: blogs/your-blog-url.com.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Exploration Corner", //required
+  "subtitle": "The Tidy Trekker", //optional
+  "type": "blog", //required
+  "url": "https://thetidytrekker.com/blog.html", //required
+  "photo_url": "https://thetidytrekker.com/assets/EC-Header.gif", //required
+  "description": "A mix of R Stats, general coding, and personal content",
+  "language": "en (required)", //required
```
Comment (drmowinckels) [inline blogs/your-blog-url.com.json:8]:
```suggestion
  "language": "en",
```

File: blogs/your-blog-url.com.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "authors": [ //required
+    {
+      "name": "Meghan Harris", //required
```
Comment (drmowinckels) [inline blogs/your-blog-url.com.json:11]:
```suggestion
      "name": "Meghan Harris",

File: blogs/your-blog-url.com.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Exploration Corner", //required
+  "subtitle": "The Tidy Trekker", //optional
+  "type": "blog", //required
+  "url": "https://thetidytrekker.com/blog.html", //required
+  "photo_url": "https://thetidytrekker.com/assets/EC-Header.gif", //required
+  "description": "A mix of R Stats, general coding, and personal content",
+  "language": "en (required)", //required
+  "authors": [ //required
```
Comment (drmowinckels) [inline blogs/your-blog-url.com.json:9]:
```suggestion
  "authors": [

**Review by drmowinckels — CHANGES_REQUESTED** [review]
Thank you for submitting!!

Can you please also change the name of the file to:
thetidytrekker.com.json

This is the best assurance we have of keeping json-filenames unique

**Review by Meghansaha — COMMENTED** [review]
accepted all and changed file name

**Pull request merged**